articles/data-factory/continuous-integration-deployment.md

```diff
@@ -55,7 +55,7 @@ Below is a sample overview of the CI/CD lifecycle in an Azure data factory that'
 
    ![Build your own template](media/continuous-integration-deployment/custom-deployment-build-your-own-template.png) 
 
-1. Select **Load file**, and then select the generated Resource Manager template.
+1. Select **Load file**, and then select the generated Resource Manager template (look for the **arm_template.json** file exported in step 1)
```

Comment:
Format: . This is the arm_template.json file located in the zip file exported in step one

Reply:
Fixed.